### DISCUSSION TITLE
Debugging why we aren't seeing snapshots in app insights - a couple questions.

### DISCUSSION THREAD

**Discussion by nzandy** [microsoft/ApplicationInsights-SnapshotCollector#11] [Q&A]
Hi there - we have a .NET core 3.1 web app deployed to an app service in azure.

Last week we were trying to get snapshots turned on to help us in future for debugging issues in production after we go live (without any luck so far).

Questions:
1) If we are not currently publishing symbol files alongside our web app , will this prevent us seeing any options in App Insights to view snapshots? [This link](https://docs.microsoft.com/en-us/azure/azure-monitor/app/snapshot-debugger#publish-symbols) seems to suggest this? If this is the case I will update our CI/CD pipeline in azure to publish symbols, but thought would double check here first.

2) [Here:](https://docs.microsoft.com/en-us/azure/azure-monitor/app/snapshot-debugger#enable-application-insights-snapshot-debugger-for-your-application 
) It states:

>  Snapshot collection is available for:
> 
> .NET Framework and ASP.NET applications running .NET Framework 4.5 or later.
> .NET Core 2.0 and ASP.NET Core 2.0 applications running on Windows.

Does this mean snapshots will not be available for our .net core 3.1 app? Or are the docs just outdated?

Thanks in advance! Really hoping to get this working next week..

**Comment by pharring** [comment]
@nzandy thanks for the questions.

The main thing you will miss when you don't have symbol is that you will not be able to see the names of parameters and local variables in the debugger view. In some cases it's not too bad -- you can often still figure things out without them.

.NET Core 3.1 is definitely supported. I'll get the docs updated.

**Reply by pharring** [reply]
(I submitted PR to fix the docs)
